spec/requests/api/subscriber/subscribers_can_write_comments_spec.rb

```diff
@@ -0,0 +1,61 @@
+RSpec.describe 'POST /api/articles/:id/comments', type: :request do
+  let(:article) { create(:article) }
+  let(:subscriber) { create(:subscriber) }
+  let(:subscriber_headers) { subscriber.create_new_auth_token }
+
+  describe 'Successfully as a subscriber' do
+    before do
+      post "/api/articles/#{article.id}/comments", params: {
+        body: 'What a great website, I have subscribed solely based on the awesome design!'
+      },
+      headers: subscriber_headers
+    end
+
+    it 'is expected to return status 201' do
+      expect(response).to have_http_status 201
+    end
+
+    it 'is expected to respond with a success message' do
+      expect(response_json['message']).to eq 'Your comment has been published'
+    end
+
+    it 'is expected to add a comment to the article' do
+      expect(article.comments.count).to eq 1
+    end
+
+    it 'is expected to create an instance of a Comment with the expected body content' do
+      expect(Comment.last['body']).to eq 'What a great website, I have subscribed solely based on the awesome design!'
+    end
+  end
+
+  describe 'Unsuccessfully as a visitor' do
+    before do
+      post "/api/articles/#{article.id}/comments", params: {
+        body: 'ITS FAKE NEWS'
```

Comment:
is it possible to write like this ('IT/S Fake News') here at the place of 'ITS FAKE NEWS'?

Reply:
It is, but these South Californian people ain't so good with spelling :3